pip_deps/requirements.txt

```diff
@@ -5,3 +5,4 @@ pygobject==3.28.2
 pyusb==1.0.2
 Adafruit_BBIO==1.1.1
 smbus-cffi==0.5.1
+boto3==1.17.112
```

Comment:
all good without adding zipfile?

Reply:
Yes, zipfile is a standard python package